pkg/apis/apps/types.go

```diff
@@ -261,7 +262,7 @@ type ControllerRevisionList struct {
 }
 
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
```

Comment:
keep this comment separated by a space so it doesn't show up in godoc

Reply:
Fixed